src/app/modules/admin-shared/unauthorized/unauthorized.component.ts

```diff
@@ -6,11 +6,17 @@ import { Component, OnInit } from '@angular/core';
   styleUrls: ['./unauthorized.component.scss']
 })
 export class UnauthorizedComponent implements OnInit {
-
+  userdetails: any;
+  username: any;
   constructor() { }
 
   ngOnInit() {
-  
+    this.userdetails =  JSON.parse(localStorage.getItem('userdetails'));
```

Comment:
get it from currentUser service

Reply:
user details we are getting form the auth service, because we are using  keycloak  to get details

Reply:
then remove this local storgae